workspaces/rbac/plugins/rbac/src/index.ts

```diff
@@ -13,6 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { unstable_ClassNameGenerator as ClassNameGenerator } from '@mui/material/className';
+
+ClassNameGenerator.configure(componentName => {
+  return componentName.startsWith('v5-')
+    ? componentName
+    : `v5-${componentName}`;
+});
+
```

Comment:
Fyi: I think we don't want push this change into upstream, right?

Reply:
Hi @christoph-jerolimov  Will this hack do anything harmful to other adopters? Without it the RBAC UI in RHDH looks bad in many places :(